core/noah_core/__init__.py

```diff
@@ -1,9 +1,8 @@
 import os
-
 from flask import Flask
-from flask_sqlalchemy import SQLAlchemy
-from flask_alembic import Alembic
+from .extensions import db, migrate
```

Comment:
could this relative import be changed to a module to avoid mixing them?

Reply:
Done, ty!